data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			choice
+				`	"Glad to see that you still have a positive view of us."`
+				`	"Well we're here to do our best to make that wish come true."`
+			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn the human language, and even some humans began to learn Hai, although it seems much more difficult for a human to speak Hai than the other way around.`
+			`	While she is telling stories, Terry walks in and motions to you. "<last>. I need someone with a ship and enough skill to not get blown up. Meet me by the landing pads when you're ready."`
+
+
+
+mission "Hai Reveal: Meet the Team: Terry"
+	name "Scout the North"
+	description "Scout <waypoints> to determine the frequency of fleets traveling around the wormhole. Return to <destination> when finished."
+	source "Hai-home"
+	waypoint "Cardax"
+	waypoint "Hassaleh"
+	waypoint "Moktar"
+	waypoint "Oblate"
+	waypoint "Rajak"
+	waypoint "Rigel"
+	waypoint "Sumar"
+	waypoint "Ultima Thule"
+	waypoint "Volax"
+	to offer
+		has "Hai Reveal: Meet the Team: Alondo: done"
+		has "Hai Reveal: Meet the Team: Xilin: done"
+		has "Hai Reveal: Meet the Team: Samuel: done"
+	on offer
+		conversation
+			`When you exit onto the landing pads, you find that Terry has just finished refueling your ship for you. "Alright, Captain. I have a simple mission for you. We need someone to scout out the ship frequencies in various uninhabited systems in the Far North. We know how many merchants are arriving here, but we're not sure how many are attempting and failing, for one reason or another. I need you to fly through these systems and report back here with what you find."`
+			`	She takes a deep breath before saying: "<waypoints>. Think you can remember all those? Good." She's didn't even wait for a response. "Stick around in each system if you can, as I want to know just how many merchants, pirates, and Navy ships are passing through that region."`
+			`	She tosses you a data pad containing coordinates to all the systems she mentioned and starts walking back inside. "Just come back here once you're finished."`
+				accept
+	on visit
+		dialog phrase "generic waypoint on visit"
+
+
+
+mission "Hai Reveal: Meet the Team: Sayari"
+	landing
+	name "Important Message"
+	description "Deliver an important message from the Hai to the Republic on <stopovers>. Return to <destination> when finished."
+	source "Hai-home"
+	stopover "Earth"
+	to offer
+		has "Hai Reveal: Meet the Team: Terry: done"
+	on offer
+		conversation
+			`Terry quickly takes a copy of your ship's system logs upon your return. "Thank you, Cpatain. I'll have these analyzed in no time." Terry runs back into the building, passing Sayari on her way out, carrying a small box.`
```

Comment:
```suggestion
			`Terry quickly takes a copy of your ship's system logs upon your return. "Thank you, Captain. I'll have these analyzed in no time." Terry runs back into the building, passing Sayari on her way out, carrying a small box.`
```
About `passing Sayari on her way out`, it would look better if it were `Terry runs back into the building carrying a small box, passing Sayari on her way out` but I could see that it might be this way to make her actions seem rushed.

Reply:
Oh I should have known that. thanks. I'll edit it.